brd-ios/breadwallet/src/Redesign/Views/Core/FELabel.swift

```diff
@@ -50,9 +53,26 @@ class FELabel: UILabel, ViewProtocol {
             
         case .attributedText(let value):
             attributedText = value
+            
+        case .attributedLink(let value, let url):
+            attributedText = value
```

Comment:
imo this adds unneded complexity to this view.
you should just add a tapGesture to the cell in ur scenes tableview and call a cycle for it